cms.css

```diff
@@ -0,0 +1,27 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
```

Comment:
I think `margin: 0` is default. Are you using `normalize`?

Reply:
No. You need to set it to 0 if not using normalize.